lib/ts/recipe/passwordless/types.ts

```diff
@@ -130,41 +134,54 @@ export type RecipeInterface = {
     }>;
 
     // TODO NEMI: Wouldnt it make more sense for this to just be a promise?
-    getLoginAttemptInfo: <CustomAttemptInfoProperties>(input: { userContext: any }) =>
+    getLoginAttemptInfo: <CustomLoginAttemptInfoProperties>(input: { userContext: any }) =>
         | Promise<
               | undefined
               | ({
                     deviceId: string;
                     preAuthSessionId: string;
-                    contactInfo: string;
-                    contactMethod: "EMAIL" | "PHONE";
                     flowType: PasswordlessFlowType;
-                    lastResend: number;
-                } & CustomAttemptInfoProperties)
+                    // contactInfo: string;
+                    // contactMethod: "EMAIL" | "PHONE";
+                    // lastResend: number;
+                } & CustomLoginAttemptInfoProperties)
           >
         | ({
               deviceId: string;
               preAuthSessionId: string;
-              contactInfo: string;
-              contactMethod: "EMAIL" | "PHONE";
               flowType: PasswordlessFlowType;
-              lastResend: number;
-          } & CustomAttemptInfoProperties)
+              //   contactInfo: string;
+              //   contactMethod: "EMAIL" | "PHONE";
+              //   lastResend: number;
+          } & CustomLoginAttemptInfoProperties)
         | undefined;
 
     // TODO NEMI: Wouldnt it make more sense for this to just be a promise?
     setLoginAttemptInfo: <CustomStateProperties>(input: {
         attemptInfo: {
             deviceId: string;
             preAuthSessionId: string;
-            contactInfo: string;
-            contactMethod: "EMAIL" | "PHONE";
             flowType: PasswordlessFlowType;
-            lastResend: number;
+            // contactInfo: string;
+            // contactMethod: "EMAIL" | "PHONE";
+            // lastResend: number;
         } & CustomStateProperties;
         userContext: any;
     }) => Promise<void> | void;
 
     // TODO NEMI: Wouldnt it make more sense for this to just be a promise?
     clearLoginAttemptInfo: (input: { userContext: any }) => Promise<void> | void;
+
+    didLoginAttemptInfoChangeAfterResend: (input: {
```

Comment:
I don't think this function is needed at all in the recipe interface. It's a simple check for `deviceId` comparison anyway + why would the user want to override this?

Reply:
We no longer need to make a device id check, removed this function from the recipe interface